extensions/wikia/VideoHandlers/VideoHandlers.setup.php

```diff
@@ -278,7 +274,6 @@
 	27 => 'Twitchtv',
 	28 => 'Ooyala',
 	29 => 'Iva',
-	30 => 'Snappytv',
```

Comment:
@lizlux @saipetch @garthwebb - I didn't know if it's ok to fix the ordering in this array. I searched the code and I didn't find any relation between Ustream and id:31 (the same for the next ones). Guys, do you know about possible side effects of changing ids in this array?